multimodal/src/autogluon/multimodal/models/mmdet_image.py

```diff
@@ -142,6 +142,7 @@ def _load_checkpoint(self, checkpoint_file):
 
         self.name_to_id = self.get_layer_ids()
         self.head_layer_names = [n for n, layer_id in self.name_to_id.items() if layer_id <= 0]
+        self.backbone_layer_names = self.get_backbone_layer_names()
```

Comment:
Instead of adding `backbone_layer_names ` and `get_backbone_layer_names `, we can have one class function to set `requires_grad` to False according to an additional hyperparameter `frozen_parts`.